### ISSUE TITLE
Possible to work on google colab?

### ISSUE THREAD

**Issue by Meddebma** [StanfordMIMI/Comp2Comp#86]
Hi, 

is it possible to run comp2comp on google colab?
and do you consider to integrate nifti format for input and output images?

Best,
Aymen

**Comment by Meddebma** [comment]
Thank you so much for this code, it worked really well. I am impressed with the quality!

**Comment by louisblankemeier** [comment]
Hi @Meddebma,

Did you get the code running on google colab? I haven't tried this so curious if it worked for you. Thank you!

**Comment by louisblankemeier** [comment]
Also, we have a function that supports nifti inputs here https://github.com/StanfordMIMI/Comp2Comp/blob/master/comp2comp/io/io_utils.py#L31 but have not yet tested this extensively as a part of the pipelines. We will at this to our todos. We do not currently support nifti outputs, besides the segmentation files which are stored as niftis.

**Comment by Meddebma** [comment]
Hi @louisblankemeier ,

yes it worked perfectly, here is the code:
`
!git clone https://github.com/StanfordMIMI/Comp2Comp/
%cd Comp2Comp 
!pip install -e .
!bin/C2C spine_muscle_adipose_tissue --input_path /content/drive/MyDrive/CT_body_comp_AI/example/sr
`
It took 90-120sec per CT scan. 
I am really impressed how good it is! 
Definitely gonna use it for my next research

**Comment by louisblankemeier** [comment]
I'm glad it worked for you on colab, that's really good to know!!

**Comment by akshaysc** [comment]
Maybe a Colab version is something we post on the repo as an example use case? Perhaps with voxel integration for viewing?

**Comment by louisblankemeier** [comment]
Agreed!! I think we've surfaced several cases where people would want to interact with outputs in python (going beyond the CLI interface) and a colab tutorial would be well suited here. As you mentioned, one case is visualizing outputs for development and QA, and voxel would be perfect for that! I think we should add this to the top of our TODOs.

**Comment by BlessingTorsu** [comment]
hello @Meddebma, 

I am trying to use Comp2Comp for my research in bariatric surgery to get the fat area. I am having some issues, and since it worked for you, I would be glad if you could help. 

bin/C2C spine_muscle_adipose_tissue --i projects/adipose_extract/sample_ct

When I run this code, I see a new directory, 'models,' created, but it has nothing.

My input file has 4 folders with an average of 98 DICOM files in each

 Which directory does the result save to? @louisblankemeier 
 
 Thank you
